common/reactive/src/test/java/io/helidon/common/reactive/WrappedOriginThreadPublisher.java

```diff
@@ -26,15 +23,15 @@
 /**
  * The WrappedOriginThreadPublisher.
```

Comment:
Class is public - the javadoc is not really describing what it is.

Reply:
This class is under `src/test`, it's used to run the reactive-stream TCK.

Reply:
oops :)